config.yaml

```diff
@@ -279,3 +315,13 @@ sdk:
           #
           # Environment variable: OTEL_LOGS_EXPORTER
           exporter: otlp
+    # Configure the log record limits. See also sdk.attribute_limits.
+    log_record_limits:
+      # Set the max log record attribute value size. Overrides sdk.attribute_limits.attribute_value_length_limit.
+      #
+      # Environment variable: OTEL_LOGRECORD_ATTRIBUTE_VALUE_LENGTH_LIMIT
```

Comment:
Can you point me to where these are specified?

Reply:
Adding them is a work in progress [here](https://github.com/open-telemetry/opentelemetry-specification/pull/2861)